src/queries.ts

```diff
@@ -44,10 +40,29 @@ export function useCurrentUser() {
   });
 }
 
-export function useSubmissionList(options: PaginationOptions) {
-  return useQuery({
-    queryKey: submissionKeys.list(options),
-    queryFn: () => nmdcServerClient.getSubmissionList(options),
+const PAGE_SIZE = 10;
+
+export function useSubmissionList() {
+  return useInfiniteQuery({
+    queryKey: submissionKeys.list(),
+    queryFn: ({ pageParam }) =>
+      nmdcServerClient.getSubmissionList({
+        limit: PAGE_SIZE,
+        offset: pageParam * PAGE_SIZE,
+      }),
+    initialPageParam: 0,
+    getNextPageParam: (lastPage, _, lastPageParam) => {
```

Comment:
I find the prefix `last` to be ambiguous here (i.e. whether it refers to the "previous" page or the "final" page).